storage-node/packages/util/ranges.js

```diff
@@ -21,18 +21,22 @@
 const uuid = require('uuid')
 const streamBuf = require('stream-buffers')
 
```

Comment:
Please, remove or uncomment all '// debug(' lines.

Reply:
done in 1b4d6d6